float64 joint_1
float64 joint_2
float64 joint_3
---
float64 x
float64 y
float64 z